repos/system_upgrade/common/actors/removeleftoverpackages/actor.py

```diff
@@ -0,0 +1,22 @@
+from leapp.actors import Actor
+from leapp.libraries.actor import removeleftoverpackages
+from leapp.models import LeftoverPackages, RemovedPackages
+from leapp.reporting import Report
+from leapp.tags import ExperimentalTag, IPUWorkflowTag, RPMUpgradePhaseTag
+
+
+class RemoveLeftoverPackages(Actor):
+    """
+    Remove packages left on the system after the upgrade to higher major version of RHEL.
+
+    Removal of packages is necessary in order to keep the machine in supported state.
+    Actor generates report telling users what packages have been removed.
+    """
+
+    name = 'remove_leftover_packages'
+    consumes = (LeftoverPackages, )
+    produces = (Report, RemovedPackages)
```

Comment:
@tomasfratrik  where is generated the report? Checking this actor, I do not see it used here.

Reply:
Checked it, and also old version from el7toel8, no reporting was done, so removing it.

Reply:
See one of the new comments, you forgot to edit the actor docstring